dask/dataframe/io/tests/test_sql.py

```diff
@@ -332,3 +335,98 @@ def test_no_character_index_without_divisions(db):
     # attempt to read the sql table with a character index and no divisions
     with pytest.raises(TypeError):
         read_sql_table("test", db, npartitions=2, index_col="name", divisions=None)
+
+
+def test_to_sql():
+    df_by_age = df.set_index("age")
+    df_appended = pd.concat([df, df,])
+
+    # Set this to False to use a sqlite DB file under the current directory, and not delete it (for easier debugging
+    # after tests have run)
+    rm_tmpdir = True
+    if rm_tmpdir:
+
+        @contextmanager
+        def tmp():
+            with tmpfile() as f:
+                yield "sqlite:///%s" % f
+
+    else:
+        from pathlib import Path
+
+        path = Path.cwd() / "tmp" / "sqlite"
+        path.parent.mkdir(exist_ok=True, parents=True)
+        uri = f"sqlite:///{path}"
+
+        @contextmanager
+        def tmp():
+            if path.exists():
+                path.unlink()
+            yield uri
+
+    for npartitions in [1, 2, 10]:
+        ddf = dd.from_pandas(df, npartitions)
+        ddf_by_age = ddf.set_index("age")
+
+        # Simple round trip test: use existing "number" index_col
+        with tmp() as uri:
+            ddf.to_sql("test", uri)
+            actual_ddf = read_sql_table("test", uri, "number").compute()
```

Comment:
Don't need to `.compute` here. `assert_eq` does that for us.

You can call this `result = `

Reply:
And use `dd.read_sql_table` like a user probably would (may have to add it to the `dask.dataframe.__init__` file.

Reply:
changed to `result` and removed `.compute()` (here and elsewhere)

sorry, I don't follow this bit:
> And use dd.read_sql_table like a user probably would (may have to add it to the dask.dataframe.__init__ file.
can you elaborate further?

Reply:
I was saying just use `dd.read_sql_table`.

You can ignore the bit about adding it to the `__init__`. I forgot that you aren't adding `read_sql_table` :)